modules/lightning_features/lightning_preview/lightning_preview.info.yml

```diff
@@ -8,3 +8,4 @@ experimental: true
 dependencies:
   - lightning_workflow
   - workspace
+components: {}
```

Comment:
SubProfileCommand is now using isset($info['components']) to check for the presence of sub-components, so things that don't have any should not define the components key at all or the command will get confused.